routes/api/profile.js

```diff
@@ -51,8 +51,17 @@ router.get('/:username', accessRouteWithOrWithoutToken, (req, res) => {
                     (user._id == req.user.id)) {
                     data.posts = posts;
                   }
-                  else { // req.user is not following OR not own post
-                   return  res.json({ msg: "This account is private. Do you want to follow?" });
```

Comment:
instead of creating a new constant  of data, u can do return  data object like  res.json(data);